.github/workflows/release.yml

```diff
@@ -15,10 +15,11 @@ jobs:
         with:
           java-version: '21'
           distribution: 'temurin'
+
+      - uses: gradle/actions/setup-gradle@v3
+
       - name: Build and Publish
-        uses: gradle/gradle-build-action@v2
-        with:
-          arguments: build publish
+        run: ./gradlew build
```

Comment:
This seems like a mistake.

Reply:
Yes, this was my oversight. I have made the necessary changes, and I have tested the `release` action.